src/allmydata/test/test_node.py

```diff
@@ -120,9 +120,8 @@ def test_tahoe_cfg_utf8(self):
         f.write(u"nickname = \u2621\n".encode('utf-8'))
         f.close()
 
-        n = TestNode(basedir)
-        n.setServiceParent(self.parent)
-        self.failUnlessEqual(n.get_config("node", "nickname").decode('utf-8'),
+        config = read_config(basedir, "")
+        self.failUnlessEqual(config.get_config("node", "nickname").decode('utf-8'),
```

Comment:
new code should use `assert...` style methods instead of the `fail...` style methods.  Up to you if you want to also make that change here or not (since this is not really a new line, just changed).